ofnetSvcProxy.go

```diff
@@ -597,6 +612,25 @@ func getInPort(pkt *ofctrl.PacketIn) uint32 {
 	return openflow13.P_ANY
 }
 
+func (proxy *ServiceProxy) findProvMac(inPort uint32, provIP net.IP) string {
```

Comment:
Can we name this function something more specific to reflect it gets the rewrite mac if both client and provider are in same subnet.